Tools/TestWebKitAPI/Tests/WebKitGLib/TestConsoleMessage.cpp

```diff
@@ -63,7 +63,11 @@ class ConsoleMessageTest : public WebViewTest {
 
     ConsoleMessageTest()
     {
+#if !ENABLE(2022_GLIB_API)
         webkit_user_content_manager_register_script_message_handler(m_userContentManager.get(), "console");
+#else
+        webkit_user_content_manager_register_script_message_handler_with_reply(m_userContentManager.get(), "console", nullptr);
+#endif
```

Comment:
We don't want to remove `webkit_user_content_manager_register_script_message_handler` from new api, we want to remove the in_world version adding the world nam parameter to `webkit_user_content_manager_register_script_message_handler`